modules/exploits/windows/local/lexmark_driver_privesc.rb

```diff
@@ -74,44 +74,48 @@ def initialize(info = {})
 
   # Check to see if a there are Lexmark drivers in the driver store.
   # If there are, validate that they are similar to the ones we want
-  # to exploit.
+  # to exploit. The user can specify the driver they'd like to exploit
+  # as option. Otherwise, the first driver is selected.
   def check
-    # List the available Lexmark drivers in the driver store. Example output:
-    # Driver name Lexmark Universal v2 XL,3,Windows x64
-    # Driver name Lexmark Printer Software G2 XL,3,Windows x64
-    # Driver name Lexmark Universal v2,3,Windows x64
-    res = cmd_exec('cmd.exe', '/c "cscript C:\\Windows\\System32\\Printing_Admin_Scripts\\en-US\\prndrvr.vbs -l | findstr /S /c:"Driver name Lexmark"')
+    res = cmd_exec('pnputil.exe /enum-drivers')
+    m = res.scan(%r{Published Name:     ([^.]*\.inf)\r\nOriginal Name:      lmu.*?.inf\r\nProvider Name:      Lexmark International\r\nClass Name:         Printers\r\nClass GUID:         {4d36e979-e325-11ce-bfc1-08002be10318}\r\nDriver Version:     (\d+)/\d+/(\d+) \d+\.\d+\.\d+\.\d+}m)
```

Comment:
My issue with this is that can't the `Class Name` technically change along with the GUID? I'm just taking a look at https://docs.microsoft.com/en-us/windows-hardware/drivers/install/system-defined-device-setup-classes-available-to-vendors and I see `PNPPrinters` and `4658ee7e-f050-11d1-b6bd-00c04fa372a7` could also be used for SCSI/1394-enumerated printers.

Other than that this looks good though :)